apps/meteor/app/2fa/server/MethodInvocationOverride.ts

```diff
@@ -2,14 +2,25 @@ import { DDPCommon } from 'meteor/ddp-common';
 import { DDP } from 'meteor/ddp';
 
 class MethodInvocation extends DDPCommon.MethodInvocation {
-	constructor(options) {
+	public twoFactorChecked: boolean;
+
+	constructor(options: {
+		connection: {
+			close: () => void;
+			clientAddress: string;
+			httpHeaders: Record<string, any>;
+		};
+		isSimulation?: boolean;
+		userId?: string;
+	}) {
 		const result = super(options);
 		const currentInvocation = DDP._CurrentInvocation.get();
 
 		if (currentInvocation) {
 			this.twoFactorChecked = currentInvocation.twoFactorChecked;
 		}
 
+		// @ts-expect-error - `super` return is typed as `void` even when types says it returns an instance of `MethodInvocation`
```

Comment:
is this something really necessary ? I think constructor always returns a instance to its right have a return void

Reply:
It should be `instance of X` for sure, but seems like an ongoing issue on TS

https://github.com/microsoft/TypeScript/issues/37847